internal/duo/duo.go

```diff
@@ -21,6 +23,7 @@ func (d *APIImpl) Call(values url.Values) (*Response, error) {
 	if err != nil {
 		return nil, err
 	}
+	logging.Logger().Tracef("Duo Auth Response Data: %s", string(responseBytes))
```

Comment:
Use the request logger to correlate logs with an actual HTTP request, this will be useful to trace the error in case Authelia is highly used.

Reply:
See how to do it in my comment below.

Reply:
Think this is resolved, fixed and pushed :)